d2l/train.py

```diff
@@ -9,19 +9,18 @@
 
 __all__ = ['evaluate_accuracy', 'train_ch3', 'train_ch5']
 
-def evaluate_accuracy(data_iter, net, device=torch.device('cpu')):
+def evaluate_accuracy(data_iter, net,device=torch.device('cpu')):
```

Comment:
keep the space between each arg, like before
```suggestion
def evaluate_accuracy(data_iter, net, device=torch.device('cpu')):
```